trade-finance-manager-api/src/graphql/resolvers/query-facilities.js

```diff
@@ -19,15 +19,23 @@ exports.queryAllFacilities = async (queryParams) => {
     // finds latest completed amendment tfm object with mapped values
     const latestCompletedAmendment = findLatestCompletedAmendment(item?.amendments);
 
+    let facilityCoverEndDate = '-';
+    let coverEndDateEpoch = '';
+    // regex to check date is correct format - YYYY-MM-DD
+    const dateRegex = /^(19|20)\d{2}-(0[1-9]|1[0-2])-(0[1-9]|1\d|2\d|3[01])$/;
```

Comment:
Can we simplify this please. Else looks good.

Reply:
done